microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -153,7 +153,8 @@ After providing the information, choose **Publish** to submit your extension to
 When your submission passes certification, your extension is published in the Microsoft Edge Add-ons store.  The status of your extension in the Partner Center dashboard changes to `In the Store`.  
 
 > [!NOTE]
-> If you encounter any issues in the submission or registration process, file a support ticket on [Extensions New Support Request][ExtensionsSupportForm] or send an email to [ext_dev_support@microsoft.com][MailtoExtDevSupportMicrosoftCom].  
+> Microsoft Edge does not support any native payment mechanism for extension monetization. You can choose to use your own suitable payment methods. Microsoft Edge does not charge any fees for any such third party payments. You get to keep the entire amount that you earn from your extension. 
```

Comment:
Suggesting some changes:

Microsoft Edge does not provide a payment mechanism for extension developers to request payments from customers. You may use a 3rd party payment provider that meets the guidelines mentioned here https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/store-policies/developer-policies#181-paid-features